app/assets/javascripts/QDMPatient.js

```diff
@@ -36,6 +35,7 @@ QDMPatientSchema.methods.initializeDataElements = function initializeDataElement
   const dataElementsInit = [];
   this.dataElements.forEach((element) => {
     typeStripped = element._type.replace(/QDM::/, '');
+    delete element._type;
```

Comment:
Let's meet in the morning to discuss this change.

Reply:
I saw this in my inbox. I suggested this change to them. It is to normalize the `_type` attribute to be what it should be to match mongoid. Once the name of the type is determined whether or not it is prepended with `QDM::`, we can remove `_type` from the object used to create the data element. This is so the default defined in the mongoose schema will be used as the `_type` and saving to the backend will be correct.